src/openrct2/localisation/Localisation.h

```diff
@@ -131,6 +131,18 @@ class Formatter
     {
         static_assert(sizeof(TSpecified) <= sizeof(uintptr_t), "Type too large");
         static_assert(sizeof(TDeduced) <= sizeof(uintptr_t), "Type too large");
+
+        // checking if the specified type is a valid one
```

Comment:
This comment more or less states what was already obvious from the code. Just get rid of it.

Reply:
Agreed